SPECS/sm.spec

```diff
@@ -419,8 +425,9 @@ cp -r htmlcov %{buildroot}/htmlcov
 %{_unitdir}/linstor-monitor.service
 
 %changelog
-* Thu Apr 29 2021 Samuel Verschelde <stormi-xcp@ylix.fr> - 2.30.3-1.1
-- WIP changelog entry
+* Thu Apr 29 2021 Ronan Abhamon <ronan.abhamon@vates.fr> - 2.30.3-1.1
+- Sync patches with our latest 2.30.3-8.2 branch
+- 0009-Fix-regression-added-by-XSI-915.patch added
```

Comment:
I would put those lines after all the CA-xxx references since the main reason for this release is "Sync with hotfix XS82E023"